### PR TITLE
Removed single project restriction.

### PR DESCRIPTION
**Pull request by jmfadeley**
Removes the restriction in place that prevents multiple projects per organization.

### CODE REVIEW

**Comment by Perry5** [comment]
Looks good! I verified that the `organizationId` wasn't a unique field in the Projects table.

File: backend/app/controllers/projects.js [hunk 1]
```diff
@@ -211,12 +211,12 @@ function* checkProjectData(req) {
...
-        );
-        if (projects.length) {
-            throw new HttpError(400, 'You can create only one project for each organization');
-        }
+        // var projects = yield thunkQuery(
+        //     Project.select().where(Project.organizationId.equals(orgId))
+        // );
+        // if (projects.length) {
+        //     throw new HttpError(400, 'You can create only one project for each organization');
+        // }
```
Comment (Perry5) [inline backend/app/controllers/projects.js]: If we have tested and it works, I would say let's delete commented out code.

Comment (sirctseb) [inline backend/app/controllers/projects.js]: Yes, delete instead of comment please

**Comment by jmfadeley** [comment]
Done.

**Pull request merged**